nav2_collision_monitor/include/nav2_collision_monitor/types.hpp

```diff
@@ -65,14 +67,16 @@ enum ActionType
   DO_NOTHING = 0,  // No action
   STOP = 1,  // Stop the robot
   SLOWDOWN = 2,  // Slowdown in percentage from current operating speed
-  APPROACH = 3  // Keep constant time interval before collision
+  APPROACH = 3,  // Keep constant time interval before collision
+  ACTION_UNKNOWN = 4 // Default for initializing action type
```

Comment:
How is this semantically different from just DO_NOTHING?

Reply:
If this is truly different, a default action should be `0` not `4` by convention and auto-initialization

Reply:
I don't see why you should publish any action if the first is `DO_NOTHING` - by definition it's not doing anything. Only just to check that its running? 

I'm not against this idea, just wanting to make sure its solving a real and not a perceived problem

Reply:
> I don't see why you should publish any action if the first is DO_NOTHING - by definition it's not doing anything.

Ah okay, I thought I should, but if that's the case then there is no need to add this new state. I was just using it to publish the first `DO_NOTHING`.